crates/red_knot_workspace/src/site_packages.rs

```diff
@@ -100,55 +244,341 @@ fn site_packages_dir_from_sys_prefix(
 
         path.push("site-packages");
         if system.is_directory(&path) {
-            tracing::debug!("Resolved site-packages directory to '{path}'");
             return Ok(path);
         }
     }
     Err(SitePackagesDiscoveryError::NoSitePackagesDirFound)
 }
 
+/// The various ways in which parsing a `pyvenv.cfg` file could fail
+#[derive(Debug, Clone, Copy, PartialEq, Eq)]
+pub enum PyvenvCfgParseErrorKind {
```

Comment:
Nit: Does the `Kind` suffix add anything useful to the name of this enum?

Reply:
It indicates that it's not an enum that implements `std::error::Error`; instead, it just stores some additional metadata providing some extra information about one of the variants of `SitePackagesDiscoveryError`. It's similar to `std::io::ErrorKind`